pull

imanwada opened the pull request.

myla17 committed:
Auto stash before merge of "master" and "origin/master"

myla17 committed:
Added documentation for preferences and notifications pages

myla17 committed:
Deleted unecessary files

myla17 committed:
Added index.html

myla17 committed:
removed "github file 2"

myla17 committed:
Added html, css, and js pages to handle preferences.

alliza committed:
Add files via upload

-> images of google and Facebook login

alliza committed:
Add files via upload

-> this is google login icon

alliza committed:
Add files via upload

-> google configuration library file

alliza committed:
Add files via upload

-> Facebook configuration library file

alliza committed:
Facebook Configuration php file

-> input your App_ID, and App secret

alliza committed:
Google Configuration file

-> Input your client Id and client Secret

alliza committed:
Schedulr login index file

-> integrated with Facebook and google login

alliza committed:
logout php file for the google login user

-> logout from schedulr

alliza committed:
logout for the facebook user 

-> logout out of schedulr

alliza committed:
Database connection

-> Save the user's information retrieved from the google and Facebook and saved in our own local host

MaeKat committed:
Started an about page

started working on an about page for us to put summaries about ourselfves and put in pictures
-added a potato place holder
-made a basic layout for about page

imanwada committed:
Merge pull request #9 from bsylve1996/calendar

Calendar

alliza committed:
Merge pull request #10 from bsylve1996/google

image

alliza committed:
Update index.php

alliza committed:
Add files via upload

Facebook login

alliza committed:
Add files via upload

google login

bsylve1996 committed:
created analog clock

created an analog clock that auto updates

bsylve1996 committed:
Merge remote-tracking branch 'origin/master'

bsylve1996 committed:
commented on file

added comment to html file for time switcher

bsylve1996 committed:
Removed unnessacry lines

myla17 committed:
Removed old js files. Currently redesigning to fit new plan

myla17 committed:
Merge remote-tracking branch 'origin/master'

myla17 committed:
Removed unecessary files

myla17 committed:
Created a lunar calendar and added code to allow month/day traversal

myla17 committed:
Edited lunar.js

myla17 committed:
Merge branch 'MMBranch'

ealvaradodev committed:
Create homepage.html

creates a html/css based homepage, only has a few lines and will be updating it some

bsylve1996 committed:
fixed time bug with 12PM

added a = to the >

bsylve1996 committed:
fixed time bug again

I added an if statement for when it is noon orginally it was if it is >= 12 no there are two if statements one fore >12 and one for ==12

ealvaradodev committed:
Update homepage.html

Added horizontal nav bar to the left column

MaeKat committed:
Merge pull request #11 from bsylve1996/about

Started an about page

MaeKat committed:
homepage and css changes

changed some html/css for homepage
changed the nav bar to fit the theme better now
Note: add margin to nav bar and see what it does with google stuff...
changed css for main css file

MaeKat committed:
centered and cliped the nav bar

centered the nave bar and clipped the edges off so it looks nicer

MaeKat committed:
made changes to hompage on about

made a mistake im afraid

MaeKat committed:
Merge pull request #12 from bsylve1996/luns

pushing homepage

MaeKat committed:
Merge branch 'master' into about

MaeKat committed:
Merge pull request #13 from bsylve1996/about

made changes to hompage on about

bsylve1996 committed:
bug fix and rearrange of code

there was an "else if if" so i updated it to "else if" and i repositioned where a line of code was in the html

ealvaradodev committed:
Update about.html

add description for imaan

imanwada committed:
Merge pull request #14 from bsylve1996/calendar

added the top nav , removed the add event , delete event buttons and the hover on the calendar. Also moved the theme button to the bottom.

myla17 committed:
Merge remote-tracking branch 'origin/master'

myla17 committed:
Edited lunar.js

myla17 committed:
Added a second lunar calendar

MaeKat committed:
changed css added two icons

added two icons 
-one for main pages
-one is for the tiny icon for the tabs
changed css for calendar to keep calendar static and not dynamically changing
also changed some spacing on the page

MaeKat committed:
Merge pull request #15 from bsylve1996/calendarccsedits

changed css added two icons

MaeKat committed:
clock css

added some css qol changes to the clock

MaeKat committed:
Merge pull request #16 from bsylve1996/timecss

clock css

ealvaradodev committed:
Update about.html

Added text and formatting, want to change font but want to make sure with team what fonts yall may want to use, because i dont want it to look bad

ealvaradodev committed:
Update about.html

Added more text into p style tags

ealvaradodev committed:
Update about.html

Added formatting to other names now

myla17 committed:
Added new and improved moon calendar

Deleted old versions

myla17 committed:
Updated mooncalendar.html

myla17 committed:
Removed unecessary "scripts" and "lunar cal" file

myla17 committed:
Merge branch 'MMBranch'

myla17 committed:
Merge remote-tracking branch 'origin/master'

myla17 committed:
Update .DS_Store

ealvaradodev committed:
Update about.html

added more to the about.

Lunsluns commented:
@bsylve1996 since there are no merge conflicts with the base branch, would you like to merge the branches?

MaeKat committed:
added updated header icon

my computer does not like moon calendar having a : so I'm a sad potato and since this is just an image I'm directly uploading it just try and stop me

ealvaradodev committed:
Add icon for Elizabeth About 

it a pic of my bun

imanwada committed:
just added the word gregorian:)

ealvaradodev committed:
Update elizabeth's icon to my jadey boi...

ealvaradodev committed:
Add apple

for Aliza

ealvaradodev committed:
Add grape

brndn

ealvaradodev committed:
add tomato

imann

ealvaradodev committed:
add tangerine

myla

ealvaradodev committed:
Update about.html

ealvaradodev committed:
Update about.html

ealvaradodev committed:
Update about.html

ealvaradodev committed:
Update about.html

alliza committed:
I committed test demo

alliza committed:
FinalCode
